scipy/stats/tests/test_continuous.py

```diff
@@ -456,11 +473,12 @@ def check_dist_func(dist, fname, arg, result_shape, methods):
     # Mean can be 0, which makes logmean -inf.
     if fname in {'logmean', 'mean', 'logskewness', 'skewness'}:
         tol_override = {'atol': 1e-15}
-    elif fname in {'mode'}:
+    elif (fname in {'mode'}) or (fname in {'median'}
+                                 and dist.__class__ in circular_families):
         # can only expect about half of machine precision for optimization
         # because math
         tol_override = {'atol': 1e-6}
-    elif fname in {'logcdf'}:  # gh-22276
+    elif fname in {'logcdf', 'ccdf'}:  # gh-22276 and such
```

Comment:
Oops. `ccdf` needs an `atol` of `2e-7` to fix the Accelerate failure. But it does seem to be the same sort of thing as gh-22276 - just a quirk of `tanhsinh` that `hypothesis` exploits to find a counterexample. There are these tiny regions in the argument space for which the error estimate is not conservative, so we end up with an inaccurate solution. If the tolerance were set tighter or if we bumped up `minlevel`, we could avoid this particular counterexample, but this could cause other problems (e.g. failure to converge to required tolerance, unnecessary iterations).

No need to re-run CI right now; I'll commit this later.